example/package.json

```diff
@@ -1,32 +1,34 @@
 {
   "scripts": {
-    "start": "react-native start",
+    "start": "expo start --dev-client",
     "android": "react-native run-android",
     "ios": "react-native run-ios",
     "web": "expo start --web",
     "eject": "expo eject"
   },
   "dependencies": {
-    "expo": "~40.0.0",
-    "expo-status-bar": "~1.0.3",
-    "react": "16.13.1",
-    "react-dom": "16.13.1",
-    "react-native": "~0.63.4",
+    "expo": "^43.0.0",
+    "expo-splash-screen": "~0.13.5",
+    "expo-status-bar": "~1.1.0",
+    "expo-updates": "~0.10.15",
+    "react": "17.0.1",
+    "react-dom": "17.0.1",
+    "react-native": "0.64.3",
+    "react-native-gesture-handler": "~1.10.2",
     "react-native-radar": "../",
-    "react-native-web": "~0.13.12",
-    "expo-splash-screen": "~0.8.0",
-    "expo-updates": "~0.4.0",
-    "react-native-gesture-handler": "~1.8.0",
-    "react-native-reanimated": "~1.13.0",
-    "react-native-screens": "~2.15.0",
-    "react-native-unimodules": "~0.12.0"
+    "react-native-reanimated": "~2.2.0",
+    "react-native-screens": "~3.8.0",
+    "react-native-unimodules": "~0.15.0",
+    "react-native-web": "0.17.1",
+    "react-native-safe-area-context": "3.3.2"
   },
   "devDependencies": {
-    "@babel/core": "~7.9.0",
+    "@babel/core": "^7.12.9",
     "babel-jest": "~25.2.6",
     "jest": "~25.2.6",
+    "radar-sdk-js": "^3.3.0",
     "react-test-renderer": "~16.13.1",
```

Comment:
I'm getting this on my machine on this branch:

```
tim@tjulien example % npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: example@1.0.0
npm ERR! Found: react@17.0.1
npm ERR! node_modules/react
npm ERR!   react@"17.0.1" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.13.1" from react-test-renderer@16.13.1
npm ERR! node_modules/react-test-renderer
npm ERR!   dev react-test-renderer@"~16.13.1" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /Users/tim/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/tim/.npm/_logs/2022-11-18T04_17_22_095Z-debug.log
```

Reply:
pls use `yarn` command instead of `npm install`